clouddriver-appengine/clouddriver-appengine.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
   implementation "io.spinnaker.kork:kork-config"
   implementation "io.spinnaker.kork:kork-cloud-config-server"
   implementation "io.spinnaker.kork:kork-moniker"
+  implementation "io.spinnaker.kork:kork-retrofit"
   implementation "com.netflix.spectator:spectator-api"
   implementation "com.squareup.okhttp:okhttp"
   implementation "com.squareup.retrofit:retrofit"
```

Comment:
Can this disappear now?

Reply:
no, I am using SpinnakerRetrofitErrorHandler in AppengineConfigurationProperties

Reply:
Sure, but SpinnakerRetrofitErrorHandler comes from kork-retrofit.  I'm asking about com.squareup.retrofit:retrofit.  If there are no more references to classes from there (e.g. RetrofitError), it can disappear.

Reply:
sure